### DISCUSSION TITLE
New Website

### DISCUSSION THREAD

**Discussion by dannyjacobs** [dannyjacobs/ECHO#46] [Show and tell]
Hi Everyone,
I got tired of wordpress with all its php and databases so I made a new website for this project.
[dannyjacobs.github.io/ECHO](https://dannyjacobs.github.io/ECHO)
  I haven't linked it from anywhere yet, but I think  its about ready. What do you think?

~Danny

**Comment by mrudula-gopalkrishna** [comment]
Hi Danny,

The website is looking good!
One thing to note is that the link to Chiropter build log memo is broken
and needs fixing.

~Mru
